src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/MemberExpansion.cs

```diff
@@ -108,6 +108,11 @@ internal sealed class MemberExpansion : Expansion
                 expansions.Add(staticMembersExpansion);
             }
 
+            if (value.NativeComPointer != 0)
```

Comment:
The "Native View" row will appear between "Static members" and "Non-public members". Is that the same ordering as the legacy EE?